README.md

```diff
@@ -10,7 +10,7 @@ Status: stage 2
 
 In WebIDL, possibly-ill-formed Strings are referred to using the [DOMString](https://webidl.spec.whatwg.org/#idl-DOMString) type. But for interfaces which operate on Unicode text, WebIDL also defines the [USVString](https://webidl.spec.whatwg.org/#idl-USVString) type as the set of all possible sequences of Unicode Scalar Values, which are all of the Unicode code points apart from the surrogate code points (`U+0000..U+D7FF` and `U+E000..U+10FFFF`), representing Unicode text.
 
-The WebAssembly [Component Model](https://github.com/WebAssembly/component-model) requires well-formed strings, as do some compile-to-JS programming languages, many data encodings, network interfaces, filesystem interfaces, etc. Interfacing JavaScript strings with such APIs is a common use case that therefore suffers from conversion burdens. In particular because conversion from `DOMString` to `USVString` is lossy (common options are to replace unpaired surrogates or to throw an error) there is a regular need for string validation both within the platform and for certain userland use case scenarios.
+The WebAssembly [Component Model](https://github.com/WebAssembly/component-model) has chosen to only support well-formed strings, even though it could also support `DOMString`. Unlike ECMAScript (JavaScript) and similar Java-like languages, some other programming languages, many data encodings, network interfaces, filesystem interfaces, etc. support only well-formed strings. Interfacing JavaScript strings with such APIs is a common use case that therefore suffers from conversion burdens. In particular, because conversion from `DOMString` to `USVString` is lossy (common options are to replace unpaired surrogates or to throw an error), there is a regular need for string validation both within the platform and for certain userland use case scenarios.
```

Comment:
What I ultimately would like to avoid is the use of language that one way or another indicates that the decisions in the Wasm CG are either good or bad. Ideally, just the facts should be stated, as neutral as possible. Perhaps:

> The authors of the WebAssembly [Component Model](https://github.com/WebAssembly/component-model) have, as part of the WebAssembly CG's decision-making process, established support for only Unicode text (`USVString`) on WebAssembly component boundaries, whereas JavaScript strings (`DOMstring`) are not supported.

For the same reason, I am also not too comfortable with terms like "many" and would rather try to avoid these. For instance, while one can casually say that there are "many" languages representing their strings as Unicode text, there are equally "many" that don't. Without quantifying in detail, the term doesn't really add anything but imply that there are more of the one than of the other. Will continue to think about a more neutral formulation, but perhaps the points I mentioned are already in your sense as well?

Reply:
> Ideally, just the facts should be stated, as neutral as possible. Perhaps:

Sure, that suggestion seems fine.

> For the same reason, I am also not too comfortable with terms like "many"

Terms like "many" help the committee understand the extent of the issue. We do not typically solve problems that are not experienced by "many" developers. In fact, in the most recent presentation to committee, a delegate was supportive of this proposal because "there's copies of this function all over the place", indicating there are a variety of use cases that require it.

> imply that there are more of the one than of the other.

The term "many" here does not make such an implication, and I'm sure that the committee is well aware that the vast majority of web APIs consume and produce DOMStrings, so that doesn't seem worth calling out either.

Reply:
I've rephrased more of the paragraph, trying to address the different opinions. PTAL :)